pkgs/by-name/ru/rure/update.sh

```diff
@@ -27,7 +27,7 @@ cat > pin.json <<EOF
 {
   "pname": "$crate",
   "version": "$version",
-  "hash": "$(nix hash to-sri --type sha256 "$(printf '%s' "$prefetch" | head -n1)")"
+  "hash": "$(nix-hash --type sha256 --to-sri "$(printf '%s' "$prefetch" | head -n1)")"
```

Comment:
The solution is probably to avoid the sri conversion altogether (untested).

```suggestion
  "hash": "sha256:$(printf '%s' "$prefetch" | head -n1)"
```

Reply:
hmm, the hash will be different tho

Reply:
Just the representation/printing, not the hash itself.

Reply:
Done in #398848.